client/my-sites/plugins-wpcom/plugins-list.jsx

```diff
@@ -1,35 +0,0 @@
-/**
```

Comment:
I've heavily changed `plugin-panel.jsx`, adding a whole new set of Jetpack-related components. Everything else in that folder is pretty much untouched.